docs/releases/2.2.txt

```diff
@@ -93,6 +93,11 @@ Database backends
 
 * Added result streaming for :meth:`.QuerySet.iterator` on SQLite.
 
+Forms
+~~~~~
+
+* Allow override formset order field widget.
```

Comment:
This should be moved to `3.0.txt`

I'd say something more here. Maybe...

```
Formsets may override the widget used when ordering forms by setting the `ordering_widget` attribute or overriding `get_ordering_widget()`.
```

(Linking both the attribute and method names.)